src/bundle/Resources/views/admin/content_type/draft_list.html.twig

```diff
@@ -0,0 +1,92 @@
+{% trans_default_domain 'content_type' %}
+
+{% import "@ezdesign/admin/content_type/macros.html.twig" as macros %}
+
+{% form_theme form_draft_content_types_delete '@ezdesign/form_fields.html.twig' %}
+
+{% if pager.nbResults %}
+    <section class="container mt-4 px-5">
+        <div class="ez-table-header">
+            <div class="ez-table-header__headline">
+                {{ 'content_type.view.draft_list.title'|trans({ '%identifier%': content_type_group.identifier })|desc('Draft Content Types in %identifier%') }}
+            </div>
+            <div>
+                {% if can_delete %}
+                    {% set modal_data_target = 'delete-draft-content-types-modal' %}
+                    <button
+                            id="draft-delete-content-types"
+                            type="button"
+                            class="btn btn-danger"
+                            data-toggle="modal"
+                            data-target="#{{ modal_data_target }}"
+                            title="{{ 'content_type.view.draft_list.action.delete'|trans|desc('Delete Draft Content Type') }}">
```

Comment:
Not sure about this on but shouldn't it be `Delete Content Type draft`? cc @DominikaK